server/utils/helpers/chat/convertTo.js

```diff
@@ -33,10 +35,27 @@ async function convertToJSONL(workspaceChatsMap) {
     .join("\n");
 }
 
-async function prepareWorkspaceChatsForExport() {
+async function prepareWorkspaceChatsForExport(type = "jsonl") {
```

Comment:
Do a validation check here to ensure that the type is in the `exportMap` key set.